src/goose/brokers/sqs/broker.clj

```diff
@@ -0,0 +1,82 @@
+(ns goose.brokers.sqs.broker
+  (:require
+   [goose.broker :as b]
+   [goose.brokers.sqs.worker :as sqs-worker]
+   [goose.brokers.sqs.sqs-requests :as sqs-request]
+   [cognitect.aws.client.api :as aws]
+   [goose.utils :as u]))
+
+(defprotocol Close
+  "Closes connection to SQS Message Broker."
+  (close [this]))
+
+(defrecord SQS [client opts]
+  b/Broker
+  ;; SQS supports nothing more than:
+  ;;   - Enqueue a job
+  ;;   - Start a worker process
+  ;;   - Enqueued jobs API
+  ;;   - Batch jobs (Will implement later)
+
+  (enqueue
+    [this job]
+    (let [{:keys [client]} this
+          {:keys [queue-url]} opts]
+
+      (sqs-request/enqueue client queue-url (u/encode-to-str job))))
```

Comment:
Why `encode-to-str` instead of `encode`?

Reply:
First the reason to encode was that the json/write-str used in sqs_requests.clj ignored the namespaces
So a function like `goose.core/process-job` would turn to just `process-job`(It took hours to figure this out when I was implementing it)

So encode-to-str will convert this to a simple string, that can be serialized and deserialized into json which is more readable imo